src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/pdl/PdlService.kt

```diff
@@ -214,11 +205,13 @@ class PdlService(
         )
     }
 
+    // Util:
```

Comment:
Når en fil blir så stor at man legger til kommentarer for å strukturere så kan det være et tegn på at den burde deles opp. (Må ikke)